app/assets/sass/_wmt/_wmt-data-table.scss

```diff
@@ -54,6 +54,10 @@ table.data-table tr.data-sub-header td:nth-child(7) {
     font-weight: bold;
 }
 
+table td.capacityRed {
```

Comment:
Minor: in our css class names we usually put a hyphen between the words rather than using camel case, so this would become `capacity-red`.